openedx/core/djangoapps/embargo/tests/test_views.py

```diff
@@ -85,3 +94,52 @@ def _load_page(self, access_point, message_key, expected_status=200):
                 actual=response.status_code
             )
         )
+
+
+@skip_unless_lms
+class CheckCourseAccessViewTest(CourseApiFactoryMixin, ModuleStoreTestCase):
+    """ Tests the course access check endpoint. """
+
+    @patch.dict(settings.FEATURES, {'EMBARGO': True})
+    def setUp(self):
+        super(CheckCourseAccessViewTest, self).setUp()
+        self.url = reverse('api_embargo:v1_course_access')
+        user = self.create_user("staff", is_staff=True)
+        self.client.login(username=user.username, password='edx')
```

Comment:
Don't hardcode the password. I'm pretty certain this is stored in a constant someplace. Use it.